src/deploy/NVA_build/setup_platform.sh

```
@@ -40,8 +40,7 @@ function install_supervisor {
     # Autostart supervisor
     deploy_log "setup_supervisors autostart"
     bin_supervisord=$(find / -name supervisord | grep bin)
-    mv ${bin_supervisord} /usr/bin/supervisord_orig
-    mv /tmp/supervisord /usr/bin/supervisord
+    mv ${bin_supervisord} /usr/bin/supervisord
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)